.gitignore

```diff
@@ -25,6 +25,7 @@ newrelic_agent.log
 Vagrantfile
 TODO
 *.log
+.gitattributes
```

Comment:
why did you add this to the gitignore?

Reply:
For local testing, then forgot to remove it. Will do.